Doc/whatsnew/3.6.rst

```diff
@@ -1200,7 +1200,7 @@ importlib
 
 Import now raises the new exception :exc:`ModuleNotFoundError`
 (subclass of :exc:`ImportError`) when it cannot find a module.  Code
-that current checks for ``ImportError`` (in try-except) will still work.
+that current checks for :exc:`ImportError` in try-except) will still work.
```

Comment:
I don't think that it's an issue. I prefer to always use :exc: role to always have the same formatting.

Reply:
If you want to get the same formatting, you can use ``` :exc:`!ImportError` ```. But it is more verbose.

Since the formatting is not missed, but it just don't create a link, I'm sure this was done for purpose.

Reply:
I don't why it's an issue to have two links to the same exception in the same paragraph. IMHO it's a good thing :)

Reply:
suggestions? remove/keep? tell me ;-)

Reply:
@serhiy-storchaka done, thanks